services/madoc-ts/src/frontend/site/pages/loaders/project-loader.tsx

```diff
@@ -52,9 +53,10 @@ export const ProjectLoader: UniversalComponent<ProjectLoaderType> = createUniver
                 </Button>
               </InfoMessage>
             ) : null}
-            {renderUniversalRoutes(route.routes, {
-              project,
-            })}
+            <Outlet />
+            <Slot layout="flex" name="project-footer">
+              <FooterImageGrid />
```

Comment:
Should wrap this in `AvailableBlocks` component so that it doesn't appear by default